src/main/Port.Adapter/UI/Views/Blazor.Common/wwwroot/style.css

```diff
@@ -6,7 +6,11 @@
 
 /* blazor.common - treeview */
 .blue {
-    fill: #1E90FF !important;
+    fill: #0781fc !important;
```

Comment:
Can we keep the original blue as it is? eg. 1E90FF

Reply:
Done